app/metrics.yaml

```diff
@@ -1236,6 +1236,72 @@ private_browsing_mode:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-03-01"
+  settings_shortcut:
+    type: event
+    description: >
+      A user pressed the "Add private browsing shortcut" button in settings.
+    bugs:
+      - 4658
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/5194
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  cfr_add_shortcut:
```

Comment:
I'd prefer not using an acronym here since its meaning isn't clear without looking at the description.

Reply:
I agree, but there's a character limit and I couldn't think of any other good names.

Reply:
Ah, I wasn't aware there is a limit. In that case I can't think of a better name either :/